dist/openshift/cincinnati.yaml

```diff
@@ -308,3 +309,6 @@ parameters:
   - name: RUST_BACKTRACE
     value: "0"
     displayName: Set RUST_BACKTRACE env var
+  - name: GB_UPSTREAM_PAUSE_SECS
+    value: "120"
+    displayName: Pause between upstream scrapes
```

Comment:
We should not use anonymous auth for github v3 API call. As you easily hit the rate limits if you are behind NAT networking. Github considers all calls to github from NAT network as a single source. So we should implement a way to use OAuth2 tokens for github calls. Also without oauth2 this is not ready for production use and as we are following continuous delivery we should not differentiate between stage and production.